src/index.html

```diff
@@ -9,10 +9,67 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image">
+        <a
+          href="#"
+          class="image"
+        ></a>
+      </div>
+
+      <div class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__product-code">Product code: 195434</div>
+
+      <div class="card__reviews">
+        <div class="stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+        </div>
+        <div class="reviews">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="price">Price:</div>
+        <div class="amount">$2,199</div>
+      </div>
+
+      <div class="button">
+        <a
+          class="button__buy"
+          href="#"
```

Comment:
Similar to the previous comment, the `href` attribute for this anchor tag is set to '#'. Ensure this is intentional or replace it with a valid URL.